cpp/include/cudf/io/text/data_chunk_source.hpp

```diff
@@ -92,6 +92,21 @@ class data_chunk_reader {
    */
   virtual std::unique_ptr<device_data_chunk> get_next_chunk(std::size_t size,
                                                             rmm::cuda_stream_view stream) = 0;
+
+  /**
+   * @pure @brief Copy the next chunk of bytes from the data source in to a specific destination
```

Comment:
What is `@pure`?

Reply:
No idea! It was in the other comment, so when I copied to modify, I left it in. @bdice any idea what this is?

Reply:
https://www.doxygen.nl/manual/commands.html#cmdpure
> Indicates that the member documented by the comment block is pure virtual, i.e., it is abstract and has no implementation associated with it. This command is intended for use only when the language does not support the concept of pure virtual methods natively (e.g. C, PHP 4).

Seems like this doesn't apply here.
```suggestion
   * @brief Copy the next chunk of bytes from the data source in to a specific destination
```

Reply:
> This command is intended for use only when the language does not support the concept of pure virtual methods natively (e.g. C, PHP 4).

Seems unnecessary.